### DISCUSSION TITLE
mtkime exception when downloading old-dated photos

### DISCUSSION THREAD

**Discussion by OWL16** [icloud-photos-downloader/icloud_photos_downloader#754] [General]
iCloud Photos does not allow for "unknown date" as a date for a photo. For scanned old family photos in my iCloud Photo Library, I have put the date for any photo of which I have no date yet, prior to the invention of photography ;-). In my case, 1-1-1750. So in the exifdata of the original photo, the "Date/Time Original" is 1750-01-01. 

On macOS Sonoma (and maybe most other OSses) This causes an unhandeld exception in download.py, line 35, as there it tries to set this date/time to the access and modified timestamps of the OS. As access/modified times are not that important on originals that actually have exifdata with "Date/Time Original" (as that should always be leading), I guess this section could be wrapped in an exception handler.

**Comment by AndreyNikiforov** [comment]
do you still have this issue in the latest version of `icloudpd`? I gave a light attempt to repro and it was unsuccessful (attempt).
